core/arch/asm_defines.asm

```diff
@@ -131,6 +131,7 @@
 # define RAW(n) .byte HEX(n) @N@
 # define DECLARE_FUNC_SEH(symbol) DECLARE_FUNC(symbol)
 # define PUSH_SEH(reg) push reg
+# define POP_SEH(reg) pop reg
```

Comment:
You defined POP_SEH only for one assembler and left it undefined for the others.

Reply:
Just saw that and just updated (we do't need the pop macro) I think this was the reason for the Appveyor failure.